### PR TITLE
Create basic components for home page

### LINKED ISSUE #9: View: Main
Create main view

### PR DESCRIPTION
**Pull request by troym9731**
Closes #9

### CODE REVIEW

File: src/client/Logo.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Logo = () => (
+  <div className="logo home-content__logo">
+    <span>{'{az}'}</span>
```
Comment (therealklanni) [inline src/client/Logo.js]: wat?

File: src/client/MainNav.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const MainNav = () => (
+  <nav className="primary-nav">
+      <a href="/">
+        <span className="logo">{'{az}'}</span> Web Developers
```
Comment (therealklanni) [inline src/client/MainNav.js:6]: `<Logo />` ?

File: src/client/Logo.js (hunk 1 above)
Comment (troym9731) [inline src/client/Logo.js]: Yeah, I just really wanted to abstract away the logo... Haha looking back, not the best idea

**Comment by therealklanni** [comment]
Why was this closed? No longer need it?

**Comment by troym9731** [comment]
Yeah I closed it and was thinking about something else, but we might still use it. Might've closed it too soon.

**Pull request closed without merging**